### PR TITLE
[[feat]] Fix version of packages used

### PR DESCRIPTION
**Pull request by jopime**

### CODE REVIEW

File: Python/libraries/recognizers-text/setup.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 from setuptools import setup, find_packages
 
 NAME = "recognizers-text"
-VERSION = "2.1.0-nlp"
-REQUIRES = ['emoji', 'multipledispatch']
+VERSION = "2.2.0-nlp"
+REQUIRES = ['emoji==0.5.4', 'multipledispatch==0.6.0',  'recognizers-text-date-time']
```
Comment (jopime) [inline Python/libraries/recognizers-text/setup.py]: remove recognizers-text-date-time

Comment (jopime) [reply Python/libraries/recognizers-text/setup.py]: Solved

**Review by aduqueanguera — APPROVED** [review]
LGTM

**Review by samuelrb — APPROVED** [review]
LGTM

**Pull request merged**